contracts/Core/StakeManager.sol

```diff
@@ -286,22 +310,28 @@ contract StakeManager is Initializable, StakeStorage, StateManager, Pause, Stake
 
     /// @notice Used by anyone whose lock expired or who lost funds, and want to request withdraw
```

Comment:
Please update comments on function to what the reflect now, also if possible please add on PR how this PR resolves P + e.